### PR TITLE
✔️ Minor Changes (Demo)

### PR DESCRIPTION
**Pull request by kulothunganug**
### Description of Changes
* changed `MDLabel` usage in demo

### CODE REVIEW

File: demos/fortnightly/libs/kv/root_screen.kv [hunk 1]
```diff
@@ -2,26 +2,22 @@
 #: import FortnightlyToolbar libs.baseclass.toolbar.FortnightlyToolbar
 #: import StiffScrollEffect kivymd.stiffscroll.StiffScrollEffect
 
+<FortnightlyRootScreen>:
+    MDBoxLayout:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/root_screen.kv:7]: Please make one indent after the root widget :)

File: demos/fortnightly/libs/kv/root_screen.kv [hunk 2]
```diff
@@ -2,26 +2,22 @@
...
-		FortnightlyToolbar:
-
-		ScrollView:
-			effect_cls: StiffScrollEffect
+        FortnightlyToolbar:
 
-			MDList:
+        ScrollView:
+            effect_cls: StiffScrollEffect
+            MDList:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/root_screen.kv:15]: Please make one indent after the root widget :)

File: demos/fortnightly/main.py [hunk 3]
```diff
@@ -35,12 +35,13 @@
 #:import FadeTransition kivy.uix.screenmanager.FadeTransition
 #:import FortnightlyRootScreen libs.baseclass.root_screen.FortnightlyRootScreen
 
-
 ScreenManager:
     transition: FadeTransition()
 
+
```
Comment (HeaTTheatR) [inline demos/fortnightly/main.py]: Please remove one indent after the root widget :)

File: demos/rally/libs/kv/overview_screen.kv [hunk 4]
```diff
@@ -35,7 +28,7 @@
                 bar_color: [gch("#FFCF44"), gch("#FF6859"), gch("#F8BBD0")]
 
 
-<OverviewBox>
+<OverviewBox>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/overview_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/overview_screen.kv [hunk 5]
```diff
@@ -95,7 +82,7 @@
             font_style: "Button"
 
 
-<AlertBox@MDBoxLayout>
+<AlertBox@MDBoxLayout>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/overview_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/register_screen.kv [hunk 6]
```diff
@@ -2,7 +2,7 @@
 #: import gch kivy.utils.get_color_from_hex
 
 
-<RallyRegisterScreen>
+<RallyRegisterScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/register_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/register_screen.kv [hunk 7]
```diff
@@ -50,14 +49,15 @@
...
+	        	on_release:
+	        		root.parent.current = "rally root screen"
 
 	        MDTextButton:
 	        	text: "Or Login with Touch ID"
 	        	pos_hint: {"center_x": .5}
 
 
-<RallyTextField@MDTextField>
+<RallyTextField@MDTextField>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/register_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/pie_chart_line.kv [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<PieChartLine>
+<PieChartLine>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/pie_chart_line.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/root_screen.kv [hunk 9]
```diff
@@ -13,22 +13,23 @@
 #: import RallySettingsScreen libs.baseclass.settings_screen.RallySettingsScreen
 
 
-<RallyRootScreen>
-    md_bg_color: gch("#33333d")
-
-    MDBoxLayout:
+<RallyRootScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/fortnightly/libs/kv/root_screen.kv [hunk 10]
```diff
@@ -37,61 +33,50 @@
...
             text: "WORLD"
             font_style: "Button"
-
         FortnightlyLabel:
             text: "The Quiet, yet Powerful Healthcare Revolution"
             font_style: "H5"
 
 
-<FortnightlyListItem>
+<FortnightlyListItem>:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/fortnightly/libs/kv/root_screen.kv [hunk 11]
```diff
@@ -37,61 +33,50 @@
...
                     image: f"{environ['FORTNIGHTLY_ROOT']}/assets/images/stocks.png"
-
                 FortnightlyListItem:
                     title: "TECH"
                     secondary_text: "Designers use Tech to make Futuristic Fabrics"
                     image: f"{environ['FORTNIGHTLY_ROOT']}/assets/images/fabrics.png"
 
 
-<HeadlineNews@MDBoxLayout>
+<HeadlineNews@MDBoxLayout>:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/fortnightly/libs/kv/root_screen.kv [hunk 12]
```diff
@@ -112,5 +97,5 @@
             size_hint: (None, None)
 
 
-<FortnightlyLabel@MDLabel>
+<FortnightlyLabel@MDLabel>:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/fortnightly/libs/kv/toolbar.kv [hunk 13]
```diff
@@ -1,8 +1,7 @@
 #: import environ os.environ
 #: import gch kivy.utils.get_color_from_hex
 
-
-<FortnightlyToolbar>
+<FortnightlyToolbar>:
```
Comment (HeaTTheatR) [inline demos/fortnightly/libs/kv/toolbar.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/accounts_screen.kv [hunk 14]
```diff
@@ -1,40 +1,31 @@
 #: import gch kivy.utils.get_color_from_hex
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
-
-<RallyAccountsScreen>
-
+<RallyAccountsScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/accounts_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/bills_screen.kv [hunk 15]
```diff
@@ -2,39 +2,31 @@
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
 
-<RallyBillsScreen>
-
+<RallyBillsScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/bills_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/budgets_screen.kv [hunk 16]
```diff
@@ -2,39 +2,31 @@
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
 
-<RallyBudgetsScreen>
-
+<RallyBudgetsScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/budgets_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/overview_screen.kv [hunk 17]
```diff
@@ -1,31 +1,24 @@
 #: import gch kivy.utils.get_color_from_hex
 
-
-<RallyOverviewScreen>
-
+<RallyOverviewScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/overview_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/root_screen.kv [hunk 18]
```diff
@@ -13,22 +13,23 @@
...
+    canvas:
+        Color:
+            rgba: gch("#33333d")
+        Rectangle:
+            pos: self.pos
+            size: self.size
+    BoxLayout:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/root_screen.kv:24]: Please make one indent

File: demos/rally/libs/kv/root_screen.kv [hunk 19]
```diff
@@ -65,42 +63,36 @@
...
-
             RallyBudgetsScreen:
                 name: "BUDGETS"
-
             RallySettingsScreen:
                 name: "SETTINGS"
 
-
-<ScrollView>
+<ScrollView>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/root_screen.kv [hunk 20]
```diff
@@ -65,42 +63,36 @@
...
     bar_width: 0
     effect_cls: StiffScrollEffect
 
 
-<RallyLabel@MDLabel>
+<RallyLabel@MDLabel>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/root_screen.kv [hunk 21]
```diff
@@ -65,42 +63,36 @@
...
     adaptive_height: True
 
 
-<RallyListItem>
+<RallyListItem>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/rally/libs/kv/settings_screen.kv [hunk 22]
```diff
@@ -1,6 +1,5 @@
 <RallySettingsScreen>:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/settings_screen.kv]: You don't need to put a colon after defining a rule

File: demos/shrine/libs/baseclass/product_gallery.py [hunk 23]
```diff
@@ -4,14 +4,15 @@
...
     StringProperty,
 )
+from kivy.uix.boxlayout import BoxLayout
 
 from kivymd.theming import ThemableBehavior
-from kivymd.uix.boxlayout import MDBoxLayout
 
 
-class ProductsBox(MDBoxLayout):
+class ProductsBox(BoxLayout):
```
Comment (HeaTTheatR) [inline demos/shrine/libs/baseclass/product_gallery.py]: Why aren't you using MD widgets?

File: demos/shrine/libs/baseclass/product_gallery.py [hunk 24]
```diff
@@ -27,10 +28,9 @@ class ProductsBox(MDBoxLayout):
     ]
     """
     callback = ObjectProperty()
-    _root = ObjectProperty()
 
 
-class ImageProduct(ThemableBehavior, MDBoxLayout):
+class ImageProduct(ThemableBehavior, BoxLayout):
```
Comment (HeaTTheatR) [inline demos/shrine/libs/baseclass/product_gallery.py]: Why aren't you using MD widgets?

File: demos/shrine/libs/baseclass/toolbar.py [hunk 25]
```diff
@@ -4,12 +4,12 @@
...
 from kivy.metrics import dp
 from kivy.properties import StringProperty
+from kivy.uix.boxlayout import BoxLayout
 
 from kivymd.theming import ThemableBehavior
-from kivymd.uix.boxlayout import MDBoxLayout
 
 
-class ShrineToolbar(ThemableBehavior, MDBoxLayout):
+class ShrineToolbar(ThemableBehavior, BoxLayout):
```
Comment (HeaTTheatR) [inline demos/shrine/libs/baseclass/toolbar.py]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/product_gallery.kv [hunk 26]
```diff
@@ -51,7 +51,7 @@
         height: root.box_height
         width: root.box_width
 
-        MDBoxLayout:
+        BoxLayout:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/product_gallery.kv:54]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/product_gallery.kv [hunk 27]
```diff
@@ -77,7 +77,7 @@
         size_hint: None, None
         size: root.box_width * 60 / 100, 100
 
-    MDBoxLayout:
+    BoxLayout:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/product_gallery.kv:80]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/product_screen.kv [hunk 28]
```diff
@@ -13,7 +13,7 @@
     adaptive_size: True
     on_touch_down: if self.collide_point(*args[1].pos): root.press_on_plan(root)
 
-    MDBoxLayout:
+    BoxLayout:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/product_screen.kv:16]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/product_screen.kv [hunk 29]
```diff
@@ -25,23 +25,19 @@
...
+            text: root.text_item
             halign: 'center'
             valign: 'middle'
-            text: root.text_item
-            markup: True
-            color: 0, 0, 0, 1
 
 
-<MoreInformation>
+<MoreInformation>:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/product_screen.kv:34]: You don't need to put a colon after defining a rule

File: demos/shrine/libs/kv/register_screen.kv [hunk 30]
```diff
@@ -3,15 +3,16 @@
 #:import environ os.environ
 
 
-<ShrineRegisterScreen>
+<ShrineRegisterScreen>:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/register_screen.kv:6]: You don't need to put a colon after defining a rule

File: demos/shrine/libs/kv/register_screen.kv [hunk 31]
```diff
@@ -3,15 +3,16 @@
...
+
     canvas:
         Color:
             rgba: root.theme_cls.bg_dark
         Rectangle:
             pos: self.pos
             size: self.size
 
-    MDBoxLayout:
+    BoxLayout:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/register_screen.kv]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/register_screen.kv [hunk 32]
```diff
@@ -26,7 +27,8 @@
 
         MDLabel:
             text: "SHRINE"
-            adaptive_size: True
+            size_hint: (None, None)
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/register_screen.kv]: Why aren't you using `adaptive_size`?

File: demos/shrine/libs/kv/shrine_root_screen.kv [hunk 33]
```diff
@@ -13,12 +13,18 @@
 <MagicButton@MagicBehavior+MDRaisedButton>
 
 
-<ShrineRootScreen>
-    md_bg_color: gch(colors[root.theme_cls.primary_palette]["50"])
+<ShrineRootScreen>:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/shrine_root_screen.kv]: You don't need to put a colon after defining a rule

File: demos/shrine/libs/kv/shrine_root_screen.kv [hunk 34]
```diff
@@ -34,16 +40,25 @@
...
+                rgba: 1, 1, 1, 1
+            RoundedRectangle:
+                size: self.size
+                pos: self.pos
+                radius: [(45, 45), (0, 0), (0, 0), (0, 0)]
 
         ScrollView:
 
-            MDGridLayout:
+            GridLayout:
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/shrine_root_screen.kv]: Why aren't you using MD widgets?

File: demos/shrine/libs/kv/shrine_root_screen.kv [hunk 35]
```diff
@@ -34,16 +40,25 @@
...
                 id: product_gallery
                 rows: 1
                 spacing: 10
-                adaptive_width: True
+                size_hint_x: None
+                width: self.minimum_width
```
Comment (HeaTTheatR) [inline demos/shrine/libs/kv/shrine_root_screen.kv]: Why aren't you using adaptive_width?

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: demos/rally/libs/kv/accounts_screen.kv [hunk 36]
```diff
@@ -1,7 +1,7 @@
 #: import gch kivy.utils.get_color_from_hex
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
-<RallyAccountsScreen>:
+<RallyAccountsScreen>
 	ScrollView:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/accounts_screen.kv:6]: Please add one indent

File: demos/rally/libs/kv/bills_screen.kv [hunk 37]
```diff
@@ -2,7 +2,7 @@
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
 
-<RallyBillsScreen>:
+<RallyBillsScreen>
 	ScrollView:
 	    MDList:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/bills_screen.kv]: Please add one indent

File: demos/rally/libs/kv/budgets_screen.kv [hunk 38]
```diff
@@ -2,7 +2,7 @@
 #: import PieChartLine libs.baseclass.pie_chart_line.PieChartLine
 
 
-<RallyBudgetsScreen>:
+<RallyBudgetsScreen>
 	ScrollView:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/budgets_screen.kv:7]: Please add one indent

File: demos/rally/libs/kv/budgets_screen.kv [hunk 39]
```diff
@@ -2,7 +2,7 @@
...
 	    MDList:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/budgets_screen.kv:8]: Please add one indent

File: demos/rally/libs/kv/overview_screen.kv [hunk 40]
```diff
@@ -1,6 +1,6 @@
 #: import gch kivy.utils.get_color_from_hex
 
-<RallyOverviewScreen>:
+<RallyOverviewScreen>
     ScrollView:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/overview_screen.kv:5]: Please add one indent

File: demos/rally/libs/kv/overview_screen.kv [hunk 41]
```diff
@@ -1,6 +1,6 @@
...
         MDList:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/overview_screen.kv:6]: Please add one indent

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

**Comment by kulothunganug** [comment]
@HeaTTheatR I had tried to make my fork update with original kivymd. In this process many of the files had changed, sorry :pray: for taking your time (for telling issues in code). Also thanks for your advices.

**Comment by HeaTTheatR** [comment]
@Kulothungan16 You just overwrote my pull request, which I already create all these fixes.

File: demos/rally/libs/kv/bills_screen.kv [hunk 42]
```diff
@@ -3,6 +3,7 @@
 
 
 <RallyBillsScreen>
+
 	ScrollView:
 	    MDList:
```
Comment (HeaTTheatR) [inline demos/rally/libs/kv/bills_screen.kv]: Add one indent

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

**Comment by HeaTTheatR** [comment]
@Kulothungan16 
Run `git pull`, make changes and make a new pull request

**Comment by HeaTTheatR** [comment]
@Kulothungan16 You just overwritten all my changes

**Pull request closed without merging**